manifests/crd/acceleratorprofiles.opendatahub.io.crd.yaml

```
@@ -28,27 +28,36 @@ spec:
               properties:
                 displayName:
                   type: string
+                  description: "The display name of the accelerator profile visible in the dashboard UI."
                 enabled:
                   type: boolean
+                  description: "Indicates whether the accelerator profile is visible and selectable in the dashboard UI."
                 identifier:
                   type: string
+                  description: "The resource identifier of the accelerator device. Example: nvidia.com/gpu"
                 description:
                   type: string
+                  description: "A short description of the accelerator visible in the dashboard UI."
```

Is it necessary to qualify this description with `visible in the dashboard UI`?
Also use `accelerator profile` for consistency with other descriptions.
```suggestion
                  description: "A short description of the accelerator profile."
```